apps/system/utils/fscmd.c

```diff
@@ -1387,6 +1462,10 @@ const static tash_cmdlist_t fs_utilcmds[] = {
 	{"rmdir",     tash_rmdir,     TASH_EXECMD_SYNC},
 #endif
 
+#ifndef CONFIG_DISABLE_ENVIRON
+	{"format",     tash_format,     TASH_EXECMD_SYNC},
```

Comment:
How about listing by Alphabet order? That makes TASH reduce operation for ordering on runtime.